staker/_RPC_api_stake.gno

```diff
@@ -98,30 +99,38 @@ func ApiGetRewards() string {
 		}
 
 		// find all external reward list for poolPath which lpTokenId is staked
-		for _, incentiveId := range poolIncentives[deposit.targetPoolPath] {
-			incentive := incentives[incentiveId]
+		ictvList, exists := poolIncentives.Get(deposit.targetPoolPath)
+		if !exists {
+			return
+		}
+
+		for _, ictvId := range ictvList {
+			ictv, exists := incentives.Get(ictvId)
+			if !exists {
+				return
+			}
 
-			stakedOrCreatedAt := max(deposit.stakeTimestamp, incentive.startTimestamp)
+			stakedOrCreatedAt := max(deposit.stakeTimestamp, ictv.startTimestamp)
 			now := time.Now().Unix()
 			if now < stakedOrCreatedAt {
-				continue
+				return
 			}
 
-			externalWarmUpAmount, exist := positionsExternalWarmUpAmount[tokenId][incentiveId]
+			externalWarmUpAmount, exist := positionsExternalWarmUpAmount[tokenId][ictvId]
```

Comment:
Separate getter(and setter) will be much better for nested avl datas.

Reply:
same as here: https://github.com/gnoswap-labs/gnoswap/pull/437#discussion_r1897793290

that global variable will be removed